packages/expo-video/plugin/src/withExpoVideo.ts

```diff
@@ -9,6 +33,20 @@ const withExpoVideo: ConfigPlugin = (config) => {
     return config;
   });
 
+  withAndroidManifest(config, (config) => {
+    const { manifest } = config.modResults;
+
+    if (!manifest || !manifest.application || manifest.application.length === 0) {
+      console.error("Couldn't apply the expo-video plugin for Android");
+      return config;
+    }
+
+    manifest.application.forEach(enablePiPSupport);
+
+    config.modResults.manifest = manifest;
+
+    return config;
+  });
```

Comment:
Probably just something simple like this:

```suggestion
  withAndroidManifest(config, (config) => {
    const activity = AndroidConfig.Manifest.getMainActivityOrThrow(config.modResults);
    activity.$['android:supportsPictureInPicture'] = 'true';
    return config;
  });
```

Reply:
Wow that's so much simpler! Thanks 😄